### PR TITLE
all indentions to two space as one tab

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: ddhcpd/files/lib/gluon/upgrade/400-ddhcpd-firewall [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-family='ipv6',
-dest_port='1234',
-proto='udp',
-target='ACCEPT',
+  name='mesh_ddhcpd',
+  src='mesh',
+  family='ipv6',
+  dest_port='1234',
+  proto='udp',
+  target='ACCEPT',
```
Comment (sargon) [inline ddhcpd/files/lib/gluon/upgrade/400-ddhcpd-firewall]: There is a reason why there are no indents here.

File: ddhcpd/files/lib/gluon/upgrade/500-ddhcpd [hunk 2]
```diff
@@ -2,21 +2,21 @@
 local a=require'gluon.site'
 local e=require('simple-uci').cursor()
 e:section('ddhcpd','dhcp_option','server_identify',{
-option=54,
-len=4,
-payload=a.next_node.ip4()
+  option=54,
```
Comment (sargon) [inline ddhcpd/files/lib/gluon/upgrade/500-ddhcpd]: See above.

File: ddhcpd/files/lib/gluon/upgrade/400-ddhcpd-firewall (hunk 1 above)
Comment (rubo77) [reply ddhcpd/files/lib/gluon/upgrade/400-ddhcpd-firewall]: what reason?

**Comment by rubo77** [comment]
OK like this?

maybe add some more comments in the lua files and use luasrcdiet

**Pull request closed without merging**